integrations/pom.xml

```diff
@@ -42,6 +42,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.spotbugs</groupId>
+      <artifactId>spotbugs-annotations</artifactId>
```

Comment:
shouldn't this be coming from jenkins core bom?

https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml#L54

Reply:
There is an upper bounds conflict with Warnings NG. Will add an exclusion

Reply:
No, I won't. Stapler also has a conflict. I suggest pinning it for now

Reply:
should be fixed now I think, can you re-check?